applicationconfig/client/src/main/java/com/azure/applicationconfig/models/SettingSelector.java

```diff
@@ -112,19 +114,20 @@ public String label() {
      * <p>
      * Examples:
      * <ul>
-     *     <li>If {@code label = "*"}, settings with any label are returned.</li>
-     *     <li>If {@code label = "\0"}, settings without any label are returned. (This is the default label.)</li>
-     *     <li>If {@code label = "abc1234"}, settings with a label equal to "abc1234" are returned.</li>
-     *     <li>If {@code label = "abc*"}, settings with a label starting with "abc" are returned.</li>
-     *     <li>If {@code label = "*abc*"}, settings with a label containing "abc" are returned.</li>
+     *     <li>If {@code labels = "*"}, settings with any label are returned.</li>
+     *     <li>If {@code labels = "\0"}, settings without any label are returned. (This is the default label.)</li>
+     *     <li>If {@code labels = "abc1234"}, settings with a label equal to "abc1234" are returned.</li>
+     *     <li>If {@code labels = "abc*"}, settings with a label starting with "abc" are returned.</li>
+     *     <li>If {@code labels = "*abc*"}, settings with a label containing "abc" are returned.</li>
+     *     <li>If {@code labels = "abc,def"}, settings with labels "abc" or "def" are returned.</li>
      * </ul>
      *
-     * @param label The ConfigurationSetting label to match. If the provided value is {@code null} or {@code ""}, all
+     * @param labels The ConfigurationSetting labels to match. If the provided value is {@code null} or {@code ""}, all
```

Comment:
I'm not 100% sure this statement is true. According to the [documentation ](https://msazure.visualstudio.com/Azure%20AppConfig/_git/AppConfigService?path=%2Fdocs%2Fprotocol%2Fkv.md&version=GBmaster)`if not specified it implies any label.` 
The same is not true for `""`. That will imply, bring me the keys that don't have label assigned.

Reply:
Doing a quick test in .NET, results are different when label=null and label=""

Reply:
This one is interesting as the class powers both the setting selection and revision selection. In revision selection null and empty label are treated the same but in setting selection they appear to mean different things.

Reply:
Ohhh interesting. I haven't see that in the documentation, nor in .NET. How are you getting this behavior?
I am doing revisions passing only the key and I get results for that key and any label. I do revisions with a key and an empty string in the label and only get the setting which label is not set.